### DISCUSSION TITLE
Something went wrong Try again on iOS

### DISCUSSION THREAD

**Discussion by cove** [community/community#39493] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.89.0 (83472182)
Device: iPhone 11 Pro (iOS 16.1.1)
</details>

Going to this repo fails with a something went wrong error:
https://github.com/MaterializeInc/materialize

I suspect it has something to do with logging in through Okta for the org, so some SSO permission issue, as that through a browser on iOS it works fine after going through the SSO login -- if I use browser that doesn't automatically switch to the GitHub app.

**Comment by aassa1919** [comment]
Khulna

**Comment by Evil0ctal** [comment]
I have a same issue here

https://github.com/orgs/community/discussions/39496

**Reply by aassa1919** [reply]
Mbjobb
